### DISCUSSION TITLE
使用最新的android版本，输入语言老是回退到英文状态

### DISCUSSION THREAD

**Discussion by dyzzc** [fcitx5-android/fcitx5-android#288] [Q&A | 问答] [answered]
在新打开app输入框的时候，默认不是我的双拼，默认显示英文输入法![Screenshot_2023-06-23-08-40-57-00_a9ae29159a8e197594ba1d9714a3b4cc.jpg](https://github.com/fcitx5-android/fcitx5-android/assets/34211219/a47d042a-48c5-4005-bfe3-662593c03f44)

是我的设置不对吗？

**Comment by rocka** [comment, marked as answer]
想要实现在新应用打开键盘默认为双拼，有两种办法：

1. 输入法列表中保持 1. English 2. 双拼，然后全局选项里打开“默认状态为激活”
2. 输入法列表的顺序改成 1. 双拼 2. English ，全局选项里关闭“默认状态为激活”（此选项默认为关闭）

但是已经打开过的程序的输入法状态会被记住，如果想要重置输入法状态，需要把输入法重启一下，从后台划掉，或者强行停止

**Reply by dyzzc** [reply]
原来“默认状态为激活”是这个意思，已经搞定了，谢谢！
